src/shared/constants.h

```diff
@@ -137,6 +137,9 @@ static const int ESTOP_MESSAGE_SIZE_BYTES = 1;
 static const unsigned char ESTOP_PLAY_MSG = 0;
 static const unsigned char ESTOP_STOP_MSG = 1;
 
+// Number of times to send a STOP primitive when robot is disconnected from Manual Control
+static const unsigned int NUM_TIMES_SEND_STOP = 10;
```

Comment:
why do we send STOP multiple times?

Reply:
Nima suggested this change, basically with sending 1 stop primitive it might get lost so the robot doesn't get it, so just to be sure we send it a few times to ensure a stop.